libs/feature/editor/src/lib/components/record-form/form-field/form-field-simple/form-field-simple.component.html

```diff
@@ -1,29 +1,8 @@
 <input
-  *ngIf="!isSelect"
-  [type]="inputType"
+  [type]="type"
   [readonly]="readonly"
   [placeholder]="placeholder"
-  [value]="value"
-  (valueChange)="valueChange.emit($event)"
-  class="border rounded-md p-3 w-full bg-white transition-colors"
-  [ngClass]="{
-    'border-pink-500': invalid,
-    'border-gray-200': !invalid,
-    'text-gray-600': readonly
-  }"
+  [ngModel]="value"
+  (ngModelChange)="valueChange.emit($event)"
+  class="gn-ui-text-input border-gray-300 hover:border-main"
```

Comment:
this shouldn't be added I think, the gn-ui-text-input class already handles border color and this will be the same on all input fields